uber/templates/regform.html

```diff
@@ -92,7 +93,6 @@
     $(function () {
         staffingClicked();
         if ($.field('staffing')) {
```

Comment:
That is a great question! We definitely don't want to look for a checkbox here, because if the checkbox is disabled we don't want it to fire click events (I actually don't know if a disabled checkbox will fire click events, I presume it won't, but it doesn't really matter).

So, `$.field('staffing')` will find the hidden input instead of the checkbox. That should be fine, because you can't click on a hidden input, so it effectively does nothing.